src/test/ui/parser/removed-syntax-extern-const-fn.stderr

```diff
@@ -0,0 +1,24 @@
+error: extern items cannot be `const`
+  --> $DIR/removed-syntax-extern-const-fn.rs:2:5
+   |
+LL |     const fn foo();
+   |     ^^^^^ help: try using a static value: `static`
```

Comment:
Hmm; could you look-ahead 1 token where this diagnostic is being emitted to check whether `fn` or `unsafe` follows `const` and then tweak the diagnostic when it does accordingly?